validation.go

```diff
@@ -9,7 +9,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/goadesign/goa/uuid"
+	"github.com/satori/go.uuid"
```

Comment:
Since this is used by the client as well it should also use the goa `uuid` package.